README.md

```diff
@@ -18,7 +18,7 @@ be sent to yet another third party?).
 
 For an elaborate discussion see: https://github.com/mjmlio/mjml/issues/340
 
-#### Usage
+#### Local Usage
```

Comment:
Add note here explaining:

1. After merging the PR, a new version will be automatically released and pushed to ECR with <X> version bump by default
2. Explain that engineers can choose to override release bump type byusing labels: release:major, release:minor, release:patch, norelease on their PRs :)

Reply:
I think I'd stick to convention for `v` 🤔